model/registry/db.yaml

```
@@ -274,18 +274,30 @@ groups:
         brief: The name of the primary table that the operation is acting upon, including the database name (if applicable).
         note: >
           It is not recommended to attempt any client-side parsing of
-          `db.statement` just to get this property, but it should be set if
+          `db.query.text` just to get this property, but it should be set if
           it is provided by the library being instrumented.
           If the operation is acting upon an anonymous table, or more than one table, this
           value MUST NOT be set.
         examples: ['public.users', 'customers']
         tag: tech-specific-sql
-      - id: statement
+      - id: query.text
         type: string
         stability: experimental
         brief: >
-          The database statement being executed.
-        examples: ['SELECT * FROM wuser_table', 'SET mykey "WuValue"']
+          The database query being executed.
+        examples: ['SELECT * FROM wuser_table where username = ?', 'SET mykey "WuValue"']
+        tag: db-generic
+      - id: query.parameter
+        type: template[string]
+        stability: experimental
+        brief: >
+          The query parameters, `<key>` being the parameter name, the value being the parameter value.
```

```suggestion
          The query parameters used in `db.query.text`, `<key>` being the parameter name, the value being the parameter value.
```

Not blocking, but maybe to make it clear that this is useful only in relation to `db.query.text`?